src/explore-education-statistics-admin/src/components/editable/EditableContentForm.tsx

```diff
@@ -76,20 +78,29 @@ const EditableContentForm = ({
     onIdle,
   });
 
-  const validateElements = useCallback((elements: Element[]) => {
-    let error: string | undefined;
-
-    elements.some(element => {
-      if (element.name === 'image' && !element.getAttribute('alt')) {
-        error = 'All images must have alternative (alt) text';
-        return true;
-      }
-
-      return false;
-    });
-
-    return error;
-  }, []);
+  function isInvalidImage(element: Node) {
+    return (
+      (element.name === 'imageBlock' || element.name === 'imageInline') &&
+      !element.getAttribute('alt')
+    );
+  }
```

Comment:
Normally we declare handlers inside of a function using arrow function syntax.

I think this function could probably be extracted to be outside of the component though.

Reply:
Yep, I've moved it out.